docs/user-guide/configuration.md

```diff
@@ -105,15 +106,16 @@ edit_uri: root/path/docs/
     string to disable the automatic setting.
 
 !!! warning
-    On GitHub, the default "edit" path (`edit/master/docs/`) opens the page in
-    the online GitHub editor. This functionality requires that the user have and
-    be logged in to a GitHub account. Otherwise, the user will be redirected to
-    a login/signup page. Alternatively, use the "blob" path
+    On GitHub and GitLab, the default "edit" path (`edit/master/docs/`) opens
+    the page in the online editor. This functionality requires that the user
+    have and be logged in to a GitHub/GitLab account. Otherwise, the user will
+    be redirected to a login/signup page. Alternatively, use the "blob" path
     (`blob/master/docs/`) to open a read-only view, which supports anonymous
     access.
 
-**default**: `edit/master/docs/` or `src/default/docs/` for GitHub or Bitbucket
-repos, respectively, if `repo_url` matches those domains, otherwise `null`
+**default**: `edit/master/docs/` or `src/default/docs/` GitHub, GitLab or
+Bitbucket repos, respectively, if `repo_url` matches those domains,
```

Comment:
Missing "for" before "GitHub". Although, "respectively" doesn't really work as there are two items in the first list and three in the second. Sure, users in the know will understand, but we should not assume users will know that GitHub and Gitlab share the same URL scheme. How about: 

    **default**: `edit/master/docs/` for GitHub and GitLab repos or `src/default/docs/` for a Bitbucket repo...